pkgs/by-name/ap/apriltag/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  opencv4,
+  python3,
+}:
+
+let
+  opencv4WithGtk = opencv4.override {
+    enableGtk2 = true; # For GTK2 support
+    enableGtk3 = true; # For GTK3 support
+  };
+in
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
Nit: `stdenv.mkDerivation` supports a new pattern for recursively accessing its own values. Switching to that pattern will avoid some [common pitfalls regarding recursive attribute sets](https://nix.dev/guides/best-practices#recursive-attribute-set-rec). You'll then be able to access this derivations values by prefixing them with `finalAttrs`, like this:
```nix
{
  tag = "v${finalAttrs.version}"
}
```
Don't forget the closing parenthesis at the end of the file.